st2common/st2common/openapi.yaml.j2

```diff
@@ -3999,6 +4072,16 @@ definitions:
     additionalProperties: False
   ExecutionFilters:
     type: object
+  Inquiry:
+    type: object
```

Comment:
Since this is a user facing API, we need to be more specific w/ the schema.

Reply:
This has been resolved in https://github.com/StackStorm/st2/pull/3653/commits/920e88f81777eff5d0e5dbf6f86fe21f1f814415. In short, the `Inquiry` definition in the OpenAPI spec is now equivalent to the `InquiryResponseAPI` model implemented in that commit.